projects/css-codegen/tsconfig.lib.prod.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.lib.json",
+  "angularCompilerOptions": {
+    "enableIvy": false
```

Comment:
Should we remove all of the `enableIvy` from the tsconfigs? Or is there any reason we still need it with the orginal view engine.

Reply:
Good question: Angular v9 warns against publishing libraries built with IVy, and recommend continuing using ViewEngine. Since we have never published xLayers libs on NPM before, I think we can enable IVy. We will only use those libraries with xLayers anyways. And I think we can say that we only support Angular v9 and above. WDYT?